src/getVersion.c

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <os.h>
+#include <os_io_seproxyhal.h>
+#include "hathor.h"
+#include "ux.h"
+
+// handleGetVersion is the entry point for the getVersion command. It
+// unconditionally sends the app version.
+void handleGetVersion(uint8_t p1, uint8_t p2, uint8_t *dataBuffer, uint16_t dataLength, volatile unsigned int *flags, volatile unsigned int *tx) {
```

Comment:
The app magic number should have been sent here also?

In the `getVersion` command we are expecting the version and this magic number.